modules/gcp-gke/main.tf

```diff
@@ -286,7 +286,14 @@ resource "google_container_node_pool" "primary_node_pool" {
     }
 
     # Use a conditional expression to add the taint only if the 'taint' variable is non-empty
-    taint = length(var.taints) > 0 ? var.taints : null
+    dynamic "taint" {
+      for_each = var.taints != null ? var.taints : []
```

Comment:
Our convention is to add a new line after for_each. Please update the docs if this doesn't exist.

Reply:
Please check this - https://github.com/honestbank/terraform-gcp-gke/blob/6b5164180b415cd036e4e348e119a4720c69df9a/modules/gcp-gke/main.tf#L99